nipype/interfaces/mrtrix3/utils.py

```diff
@@ -677,3 +677,86 @@ def _list_outputs(self):
         outputs = self.output_spec().get()
         outputs['out_file'] = op.abspath(self.inputs.out_file)
         return outputs
+
+
+class MRResizeInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True,
+        argstr='%s',
+        position=-2,
+        mandatory=True,
+        desc='input DWI image'
+    )
+    image_size = traits.Tuple(
+        (traits.Int, traits.Int, traits.Int),
+        argstr='-size %d,%d,%d',
+        mandatory=True,
+        desc='define the new image size for the output image. This should be '
+        'specified as a comma-separated list.',
+        xor=['voxel_size', 'scale_factor'],
+    )
+    voxel_size = traits.Tuple(
+        (traits.Float, traits.Float, traits.Float),
+        argstr='-voxel %d,%d,%d',
+        mandatory=True,
+        desc='define the new voxel size for the output image. This can be '
+        'specified either as a single value to be used for all '
+        'dimensions, or as a comma-separated list of the size for each '
+        'voxel dimension.',
+        xor=['image_size', 'scale_factor'],
+    )
+    scale_factor = traits.Tuple(
+        (traits.Float, traits.Float, traits.Float),
+        argstr='-scale %d,%d,%d',
+        mandatory=True,
+        desc='scale the image resolution by the supplied factor. This can be '
+        'specified either as a single value to be used for all '
+        'dimensions, or as a comma-separated list of scale factors for '
+        'each dimension.',
+        xor=['image_size', 'voxel_size'],
+    )
+    interp = traits.Enum(
```

Comment:
I might call this `interpolation`. There's not a big advantage to shortening these variables, and it makes the help text more readable to use full words.

```suggestion
    interpolation = traits.Enum(
```